src/store/cockpitApi.ts

```diff
@@ -0,0 +1,24 @@
+import { emptyCockpitApi as api } from './emptyCockpitApi';
+import {
+  GetBlueprintsApiArg,
+  GetBlueprintsApiResponse,
+} from './imageBuilderApi';
+
+// Injects API endpoints into the service for querying blueprints
+export const blueprintsReducer = api.injectEndpoints({
```

Comment:
This looks good, but we need to teach it how to actually return a GetBlueprintsApiResponse object.

You'll want to use a queryFn, check the RTKQ docs here:
https://redux-toolkit.js.org/rtk-query/usage/customizing-queries#customizing-queries-with-queryfn

You might want to use the Cockpit Files API to read in the blueprints from a directory, docs here: https://cockpit-project.org/guide/latest/cockpit-file

We haven't 100% decided on the directory yet, but perhaps `~/.local/share/image-builder/blueprints`.